### ISSUE TITLE
Pause Menu-like Interface

### ISSUE THREAD

**Issue by justalemon** [justalemon/LemonUI#15]
This is probably the continuation of Guad/NativeUI#44.

Implementation of a UI that kinda looks like the pause menu. I remember seeing it on an open source RagePluginHook mod but I don't remember which one it was.

**Comment by RelaperCrystal** [comment]
It's RageNativeUI

**Comment by justalemon** [comment]
Not quite. RageNativeUI extended the Pause Menu Interface, but it was originally made as part of the original NativeUI by Guad.

**Comment by justalemon** [comment]
Pictures originally posted by @SkyZ0347 on #39

![](https://user-images.githubusercontent.com/87867345/131262920-f3e7a598-37fd-42cb-b05b-7e543756d3a8.jpg)
![](https://user-images.githubusercontent.com/87867345/131263016-64bbbdc4-488c-4443-8a8a-963b752c9aee.png)

**Comment by justalemon** [comment]
> I think @Eddlm implemented the scaleform once as part of his mods. Ill probably ask him in the 5mods server about what info he has about the "session" scaleform (second picture).

I was wrong, it wasn't Eddlm: https://www.gta5-mods.com/scripts/community-races
